apps/prairielearn/src/pages/partials/issueBadge.ejs

```diff
@@ -1,9 +1,9 @@
 <% if (count > 0) { %>
     <% if (!(typeof suppressLink != 'undefined' && suppressLink)) { %>
         <% if (typeof issueQid != 'undefined') { %>
-            <a class="badge badge-pill badge-danger" href="<%= urlPrefix %>/course_admin/issues?q=is%3Aopen+qid%3A<%= issueQid %>">
+            <a class="badge badge-pill badge-danger" aria-label="Assessment Issues" href="<%= urlPrefix %>/course_admin/issues?q=is%3Aopen+qid%3A<%= issueQid %>">
```

Comment:
I don't think this is an accurate label, as it's not guaranteed to describe the issues associated with an assessment. This should probably be something like `### open issues`?

Reply:
Updated label to "Open Issues" in [3b5665b](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/3b5665b6bc0be4706a91dfd033bf10fb8b256e2e).

Reply:
Sorry, by `### ...` I meant that the label should include the number of open issues, e.g. `1 open issue` or `10 open issues`.

Reply:
Ah, that makes sense. Added the number of open issues in [cfc158d](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/cfc158d8e61552bd2e4f6153413a40d61a7142b9).